validator/src/main.rs

```diff
@@ -81,34 +81,6 @@ fn hash_validator(hash: String) -> Result<(), String> {
         .map_err(|e| format!("{:?}", e))
 }
 
-fn get_shred_rpc_peers(
-    cluster_info: &ClusterInfo,
-    expected_shred_version: Option<u16>,
-) -> Vec<ContactInfo> {
-    let rpc_peers = cluster_info.all_rpc_peers();
-    match expected_shred_version {
-        Some(expected_shred_version) => {
-            // Filter out rpc peers that don't match the expected shred version
-            rpc_peers
-                .into_iter()
-                .filter(|contact_info| contact_info.shred_version == expected_shred_version)
-                .collect::<Vec<_>>()
-        }
-        None => {
-            if !rpc_peers
```

Comment:
This logic is too heavy handed now that the entrypoint advertises a proper shred version.